packages/wonder-blocks-breadcrumb/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import Breadcrumb from "./components/breadcrumb.js";
+
+import BreadcrumbItem from "./components/breadcrumb-item.js";
+
+export {Breadcrumb as default, BreadcrumbItem};
```

Comment:
I'm not actually sure we want Breadcrumb to be exported as default here. If we do this then devs will have to do: `import Breadcrumb, {BreadcrumbItem} from "...";` every time they use it. I think we should instead just export them both by name.

Reply:
In general I think most of our packages are set up such that there's either a default export or there's multiple named exports but not both.  I believe the icon package is the only exception to this and we should probably rectify that at some point.